ci/docker-rally.sh

```diff
@@ -4,6 +4,9 @@
 XTRACE=$(set +o | grep xtrace)
 set -o xtrace
 
+# Track failures
+FAILED=0
```

Comment:
It's mainly used to detect if one fails. If run fails, we want to ensure we run cleanup, but we need to remember run failed so we can return that to travis-ci. Make sense?